.circleci/config.yml

```diff
@@ -257,6 +257,27 @@ jobs:
               --junitxml=test-reports/junit.xml \
               --server http://localhost:8000
 
+  containerized-tests:
+    machine:
+      docker_layer_caching: true
```

Comment:
Since the tests always start with `make kill`, does Docker layer caching actually help us here? It has a non-trivial cost, so I'd prefer to not use if it isn't needed.

Reply:
Set to false

Reply:
@mythmon Yeah, it's two different things.  `make kill` doesn't remove the cached image layers, it just removes the running instances and their volumes.  `docker system prune` removes the cached image layers.  So caching would still help here, but only if you want it to be really fast.